Microsoft.Teams.Apps.Scrum/Bots/ScrumBot.cs

```diff
@@ -32,27 +34,48 @@ namespace Microsoft.Teams.Apps.Scrum.Bots
     /// </summary>
     public class ScrumBot : TeamsActivityHandler
     {
+        /// <summary>
+        /// Sets the team members cache key.
+        /// </summary>
+        private const string TeamMembersCacheKey = "teamMembersCacheKey";
+
         private static AsyncRetryPolicy retryPolicy = Policy.Handle<HttpOperationException>()
             .WaitAndRetryAsync(Backoff.DecorrelatedJitterBackoffV2(TimeSpan.FromMilliseconds(1000), 5));
 
+        private static AsyncRetryPolicy memberRetryPolicy = Policy.Handle<HttpOperationException>(ex => ex.Response.StatusCode == HttpStatusCode.TooManyRequests)
+            .WaitAndRetryAsync(Backoff.DecorrelatedJitterBackoffV2(TimeSpan.FromMilliseconds(1000), 5));
+
         private readonly string expectedTenantId;
         private readonly IConfiguration configuration;
         private readonly IScrumProvider scrumProvider;
         private readonly TelemetryClient telemetryClient;
 
+        ///// <summary>
+        ///// Sends logs to the Application Insights service.
```

Comment:
Please delete instead of leaving this commented out.

Reply:
Removed comments